### ISSUE TITLE
[Chore] Remove the secret in the code

### ISSUE THREAD

**Issue by longnd** [dungtruongtien/kewe-crawler#25]
Issue
--
Secrets shouldn't be hardcoded and tracked on Git
https://github.com/dungtruongtien/kewe-crawler/blob/f6ac394ae6e746bde81338710936adc800dc3af4/api-service/src/common/constant.js#L1-L2

**Comment by dungtruongtien** [comment]
Agreed. These secrets should move to environment variables (applied when building, deploy..) or somewhere safer. Should not be placed directly in the codebase, my big mistake.
